compiler/src/iree/compiler/Codegen/Common/TileSizeSelection.cpp

```diff
@@ -55,18 +55,16 @@ TilingConfig::TilingConfig(IREE::Codegen::LoweringConfigAttr lc)
 /// Returns the tiling level that contains the vector dim at `dimPos` (which is
 /// an index into the result of `getVectorTileSizes()`).
 std::optional<unsigned>
-TilingConfig::getTilingLevelForVectorDimPosition(unsigned dimPos) {
+TilingConfig::getTilingLevelForVectorDimPosition(unsigned dimPos) const {
```

Comment:
Should we made this method private? The main point of having this class is to abstract away the implementation details (specific levels in this case) so that clients don't have to reason/be aware of them.

Reply:
Sure, makes sense :+1: